_posts/en/newsletters/2020-07-22-newsletter.md

```diff
@@ -0,0 +1,195 @@
+---
+title: 'Bitcoin Optech Newsletter #107'
+permalink: /en/newsletters/2020/07/22/
+name: 2020-07-22-newsletter
+slug: 2020-07-22-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter links to several discussions about activating
+taproot and summarizes a proposed update to BIP173 bech32 addresses.
+Also included are our regular sections summarizing interesting changes to
+services and client software, releases and release candidates, and notable
+changes to popular Bitcoin infrastructure software.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Taproot activation discussions:** several discussions were started
+  or continued this week about choosing a method for activating
+  [taproot][topic taproot].
+
+    - **New IRC room:** Steve Lee [posted][lee irc] to the Bitcoin-Dev
+      mailing list an announcement of a new `##taproot-activation` IRC
+      room on the Freenode.net network.  The channel is [logged][irc
+      log] and saw an impressive amount of conversation from dozens of
+      participants during its first week.
+
+    - **Mailing list thread:** Anthony Towns posted a summary of
+      the recently-updated [BIP8][] activation method (see [Newsletter
+      #104][news104 bip8]) and of Towns's own new [bip-decthresh][]
```

Comment:
replace "Towns's" with "his", which would be unambiguous and sounds slightly more natural.